### ISSUE TITLE
AdminConnection install() and start() fail

### ISSUE THREAD

**Issue by bestbeforetoday** [hyperledger/composer#2700]
Driving the `AdminConnection` API from a business network unit test, using `deploy()` works but using `install()` and `start()` fails with the following error on the call to `start()`:
```
Error: Failed to obtain credentials for PeerAdmin: Failed to load connection profile embedded. Error was Error: ENOENT: no such file or directory, open '/Users/mark/.composer-connection-profiles/embedded/connection.json'
      at connectionProfileManager.getConnectionManager.then.then.catch (node_modules/composer-admin/lib/adminconnection.js:961:31)
```

Note the error above was connecting using a card with user name of _PeerAdmin_ and a connection profile with the name _embedded_. The error occurs in `exportIdentity()` called from `_buildStartTransaction()`.

Guess is that `AdminConnection.start()` is not setting the card into the start options used to build the start transaction. The fact that the code is looking in `.composer-connection-profiles` when using cards is a significant problem; hopefully just a follow-on error from the previous problem.

I notice that `_buildStartTransaction()` grabs credentials from the card. This might be an issue if the card does not have credentials but has been enrolled already, as I don't think we (currently) store credentials back to the card after enrollment.

**Comment by mbwhite** [comment]
Passing in a ` { card : cardInstance }` to start resolved the issue for me within the embedded test case.

**Comment by mbwhite** [comment]
Adding this code to the `adminconnection.js` before calling `_buildStartTransaction(..)` in `admincConnection.start(..)`

```
        // a card should exist in the security context from the connect call
        // use this in the start options
        startOptions.card = this.securityContext.card;
```

**Comment by mbwhite** [comment]
Resolved by #2725
